pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -1066,16 +1070,27 @@ struct ArgSortOpConversion : public OpConversionPattern<tile::ArgSortOp> {
       // unsorted region, by setting the insertion point after its block.
       rewriter.setInsertionPointAfter(minLoop);
 
-      // Swap the sort position with the minimum position.
       auto finalMinPos =
           rewriter.create<LoadOp>(loc, minIdxVar, zeroIndex).getResult();
       ops[axis] = finalMinPos;
       auto finalMinIdx = rewriter.create<LoadOp>(loc, result, ops).getResult();
+
+      // Move every elements between [sortIV,finalMinPos) a step forward
```

Comment:
Could rephrase this as: "Move all elements ..." or "Move every element ..."

Reply:
Changed to "Move every element ...".